File: asAbasDePericles.cpp
//https://judge.beecrowd.com/pt/problems/view/2061
#include <bits/stdc++.h>
using namespace std;
 
int main() {
 
    int n, m; cin >> n >> m;
    
    for(int i = 0; i < m; i++){
        string s;
        cin >> s;
        if(s == "fechou"){
            n++;
        }else{
            n--;
        }
    }
 
    cout << n << endl;

    return 0;
}

File: OBI/asAbasDePericles.py
#https://judge.beecrowd.com/pt/problems/view/2061
n, m = input().split()

n = int(n)
m = int(m)

for i in range(m):
    temp = input()

    if(temp == "clicou"):
        n -= 1
    else:
        n += 1


print(n)